bindata/v4.1.0/kube-apiserver/check-endpoints-clusterrolebinding-node-reader.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:openshift:controller:kube-apiserver-check-endpoints-node-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:node-reader
```

Comment:
link to where this is defined? I don't recognize it.

Reply:
```
Name:         system:node-reader
Labels:       kubernetes.io/bootstrapping=rbac-defaults
Annotations:  rbac.authorization.kubernetes.io/autoupdate: true
PolicyRule:
  Resources      Non-Resource URLs  Resource Names  Verbs
  ---------      -----------------  --------------  -----
  nodes/stats    []                 []              [create get]
  nodes          []                 []              [get list watch]
  nodes/metrics  []                 []              [get]
  nodes/spec     []                 []              [get]
```

I found this: https://github.com/openshift/openshift-apiserver/blob/ed260ac7ae860d493187963ce23efabb36a2884f/pkg/cmd/server/bootstrappolicy/policy.go#L411-L434

Reply:
> I found this: https://github.com/openshift/openshift-apiserver/blob/ed260ac7ae860d493187963ce23efabb36a2884f/pkg/cmd/server/bootstrappolicy/policy.go#L411-L434

this repo cannot depend on openshift-apiserver-operator like that.

Reply:
create a second clusterrole if need be